### ISSUE TITLE
VST3 validator crash - Output queue missing - ASAN

### ISSUE THREAD

**Issue by 0ax1** [free-audio/clap-wrapper#127]
For one of the VST3 validator tests, no output queue is provided, such that the wrapper crashes on https://github.com/free-audio/clap-wrapper/blob/next/src/detail/vst3/process.cpp?plain=1#L701 with `_vstdata->outputParameterChanges` being `nullptr`. This report/backtrace was generated when running the validator and wrapper compiled with [ASAN](https://clang.llvm.org/docs/AddressSanitizer.html).

Backtrace:
```
#0	0x0000000106730538 in Clap::ProcessAdapter::enqueueOutputEvent(clap_event_header const*) at clap-wrapper/src/detail/vst3/process.cpp:701
#1	0x00000001067248d0 in Clap::ProcessAdapter::output_events_try_push(clap_output_events const*, clap_event_header const*) at clap-wrapper/src/detail/vst3/process.cpp:456
#2	0x00000001062ace94 in apply_parameter_changes_to_engine at Clap/Clap.m:77
#3	0x00000001062ac96c in plug_process at Clap/Clap.m:558
#4	0x000000010672998c in Clap::ProcessAdapter::process(Steinberg::Vst::ProcessData&) at clap-wrapper/src/detail/vst3/process.cpp:397
#5	0x0000000106758e60 in ClapAsVst3::process(Steinberg::Vst::ProcessData&) at clap-wrapper/src/wrapasvst3.cpp:109
#6	0x00000001002c8c70 in Steinberg::Vst::ProcessTest::run(Steinberg::ITestResult*) at public.sdk/source/vst/testsuite/processing/process.cpp:95
#7	0x000000010000e0cc in Steinberg::Vst::Validator::runTestSuite(Steinberg::Vst::TestSuite*, char const*) at public.sdk/samples/vst-hosting/validator/source/validator.cpp:742
#8	0x000000010000ee3c in Steinberg::Vst::Validator::runTestSuite(Steinberg::Vst::TestSuite*, char const*) at public.sdk/samples/vst-hosting/validator/source/validator.cpp:802
#9	0x000000010000ee3c in Steinberg::Vst::Validator::runTestSuite(Steinberg::Vst::TestSuite*, char const*) at public.sdk/samples/vst-hosting/validator/source/validator.cpp:802
#10	0x0000000100012364 in Steinberg::Vst::Validator::testModule(std::__1::shared_ptr<VST3::Hosting::Module>, Steinberg::Vst::Validator::ModuleTestConfig const&) at public.sdk/samples/vst-hosting/validator/source/validator.cpp:563
#11	0x000000010000c208 in Steinberg::Vst::Validator::run() at public.sdk/samples/vst-hosting/validat
or/source/validator.cpp:437
```

**Comment by baconpaul** [comment]
solution here is just an if(_v->outq) {} around that block right? Nothing else?

**Comment by 0ax1** [comment]
all tests running sanitizers are passing now.

**Comment by defiantnerd** [comment]
Merged to 0.6.0
